cookbook/Makefile

```diff
@@ -2,6 +2,7 @@
 
 include common/Makefile
 include common/parent.mk
+include common/leaf.mk
```

Comment:
remind me what's the use-case for this?  like, in what scenario do we call make commands defined here.

Reply:
the `PIP_COMPILE` (defined in leaf.mk) needs to be called here to update dev/doc requirements